android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -27,7 +27,7 @@ class CustomerIOReactNativeModule(
         val isDebuggable = 0 != reactContext.applicationInfo.flags and
                 ApplicationInfo.FLAG_DEBUGGABLE
         CustomerIOReactNativeInstance.setLogLevel(
-            logLevel = if (isDebuggable) CioLogLevel.DEBUG else CioLogLevel.NONE,
+            logLevel = if (isDebuggable) CioLogLevel.DEBUG else CioLogLevel.ERROR,
```

Comment:
There are 4 types of log levels - info, none, error & debug. This condition satisfies only two of them.

Reply:
Yes, we do have 4. These log types are handled in `TypeConversion.kt`. This method just sets log based on build type until SDK is initialized. As soon as the SDK is initialized, `logLevel` is updated to match user preference 🤓